linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/ignition/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+# platform = multi_platform_ocp
```

Comment:
These are still needed for RHEL worker nodes. Trying to be a good upstream OKD steward and added Fedora too. The ignition scripts won't get added to the RHEL and Fedora datastreams without these lines.

Reply:
Thank you, makes sense. Fixed.